app/src/main/res/layout/language_items.xml

```diff
@@ -33,7 +39,7 @@
       android:layout_height="wrap_content"
       android:layout_marginEnd="24dp"
       android:fontFamily="sans-serif"
-      android:text="@{languageString}"
+      android:text="@{viewModel.language}"
       android:textColor="@color/oppiaPrimaryText"
```

Comment:
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Also, does this change needs to be in `landscape` file too?

Reply:
Done